html/semantics/forms/the-object-element/object-setcustomvalidity.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE HTML>
```

Comment:
We should not create the-object-element/ directory under semantics/forms/.  It should be in semantics/embedded-content/.  The directory structure in html/ should be matched to the structure of the HTML standard.

Reply:
Updated